static/js/stream_settings_ui.js

```diff
@@ -428,6 +428,22 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+// params {isEmpty: Booliean}
+export function render_on_empty_streams(params) {
```

Comment:
Can't we directly pass a single parameter to this function `is_empty`, instead of passing an object with a single key.